src/basic_recipes/series.jl

```diff
@@ -11,7 +11,7 @@ If any of `marker`, `markersize`, `markercolor`, `strokecolor` or `strokewidth`
 """
 @recipe Series (curves::AbstractVector{<:Union{BezierPath,AbstractVector{<:Point}}},) begin
     linewidth=2
-    color=:lighttest
+    color=wong_colors()
```

Comment:
`inherit(:palette, wong_colors())` should do it, I just don't know what the sequence of keys is in the palette to get to the colors of lines.  Maybe `inherit((:palette, :colors), wong_colors())`?  I can't remember if I added this syntax though or what the palette looks like.